src/components/Form.js

```diff
@@ -1,47 +1,85 @@
 import { useForm } from 'react-hook-form';
 import styled from 'styled-components';
 import { useImmer } from 'use-immer';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { DateRangeInput } from '@datepicker-react/styled';
 import { useReducer } from 'react';
 import { ThemeProvider } from 'styled-components';
+import MapboxGeocoder from '@mapbox/mapbox-gl-geocoder';
+import './Form.css';
+import mapboxgl from 'mapbox-gl';
 
 export default function Form({
   ButtonName,
   formName,
   initialState,
-  initialCount,
   destination,
   submit,
   preloadedValues,
 }) {
+  mapboxgl.accessToken =
+    'pk.eyJ1Ijoiai1tYXNzbWFubiIsImEiOiJjbDAyYzl5MHUwNW4yM2xxYmNrY3ViMmQ3In0.OFJuoFCLObuzC-2xN6gzZA';
```

Comment:
❗❗❗  
I don't think this token should ever be in your frontend code, let alone in GitHub. Use `.env` variables for this. And I would recommend creating a new token though. Because now it's saved in your git history and everyone can use your token.

Reply:
Oh yes, you are right!!
I will quick-fix this